### DISCUSSION TITLE
How to turn off default prompt?

### DISCUSSION THREAD

**Discussion by Menober** [lllyasviel/Fooocus#3559] [Q&A] [answered]
I think default prompt messes with my desired output - is there a way to turn off default prompt?

", very intricate, elegant, highly detailed, light, sharp focus, dramatic, beautiful, artistic, fine detail, pretty, confident, dynamic, attractive, draped, deep background, professional, cinematic, surreal, healing, magic, scientific, winning, perfect, smart, color, illuminated, determined, elite, striking, full, nice"

**Comment by Rogala** [comment, marked as answer]
"Default Promt" is made up of selected styles, each style has its own positive and negative promt.
You can adjust them as you like.

![image](https://github.com/user-attachments/assets/3b6a1ff2-1280-4dc9-9c67-f41a93d40f95)

Fooocus\sdxl_styles\
